test/functional/commands/find-e2e-specs.js

```diff
@@ -141,4 +141,85 @@ describe('find elements', function () {
       })).should.eventually.exist;
     });
   });
+
+  describe('by view matcher', function () {
+    it('should find an element using view matcher', async function () {
+      await driver.elementByXPath("//*[@text='Content']").click();
+      await driver.elementByXPath("//*[@text='Storage']").click();
+      await driver.elementByXPath("//*[@text='External Storage']").click();
+      await driver.element('-android viewmatcher', JSON.stringify({
+        name: 'withText',
+        args: 'Picture getExternalFilesDir',
+        class: 'androidx.test.espresso.matcher.ViewMatchers'
+      })).should.eventually.exist;
+      await driver.back();
```

Comment:
please do not make the test dependent on each other. failure of the first test will make all the following to fail